cli/cmd/start/start.go

```diff
@@ -28,6 +28,8 @@ func StartCmd(ver version.Version) *cobra.Command {
 				return err
 			}
 
+			defer app.Close()
```

Comment:
Does this trigger a graceful shutdown?

Reply:
It won't trigger it, but will run after a graceful shutdown. The call to `app.Close` will only run after `app.Serve` returns, and `app.Serve` will only return when the server stops (either due to a catastrophic error or a graceful shutdown)